### DISCUSSION TITLE
Can i make consult-outline search (always) only in level 2 headlines?

### DISCUSSION THREAD

**Discussion by gety9** [minad/consult#921] [Q&A] [answered]
Title. I'd like to set it permanent in buffer (or if not possible, then for all buffers). So i won't need additional parameters (like C-u 2 M-x consult-outline ret OR consult-outline > consult-narrow 2)

**Comment by minad** [comment, marked as answer]
That's not possible directly, but you can create your own command which for example passes a buffer local variable as argument to `consult-outline`.
